musica/micm/micm.meta

```diff
@@ -39,6 +39,12 @@
   type = real | kind = kind_phys
   dimensions = ()
   intent = in
+[ log_output_unit ]
+  standard_name = do_log_output
+  units = flag
+  type = logical
+  dimensions = ()
+  protected = True
```

Comment:
I think you accidentally grabbed the metadata for `do_log_output` instead of `log_output_unit`:

```suggestion
  standard_name = log_output_unit
  units = 1
  type = integer
  dimensions = ()
  protected = True
```